database/migrations/2021_03_10_100000_create_profiles_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Tipoff\Profiles\Models\ProfileLink;
+
+class CreateProfilesTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('profiles', function (Blueprint $table) {
+            $table->id();
+            $table->morphs('profileable');      //  User, Location, Company
+            $table->string('bio');
+            $table->foreignIdFor(ProfileLink::class);
```

Comment:
This should be the other way. It should be a one-to-many relationship with profile links where a profile can have many links, so the profile_id should be on the profile_links table.

Reply:
Ah, you're right. I'll fix that